api/apps/services/file_api_service.py

```
@@ -556,7 +557,7 @@ def _move_entry_recursive(source_file_entry, dest_folder_entry, override_name=No
                     new_location,
                 )
             except Exception as storage_err:
-                raise RuntimeError(f"Move file failed at storage layer: {str(storage_err)}")
+                raise RuntimeError(f"Move file failed at storage layer: {storage_err!s}")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)